EmailageJavaScriptWrapper/EmailValidator.js

```diff
@@ -0,0 +1,140 @@
+function emailValidate(requestObject, accountSId, authToken, baseUrl, requestType, dataType, sigMethod, nonce, version) {
+    if (baseUrl == undefined || (baseUrl.length) > 0 == false) {
```

Comment:
!baseUrl 
???
https://stackoverflow.com/questions/6003884/how-do-i-check-for-null-values-in-javascript#6003920

Reply:
you mean if(!baseUrl), right? There isn't a use for triple question mark. There is a triple !!!, but a single bang will do what I need here: https://stackoverflow.com/questions/19550294/what-does-the-syntax-mean-in-javascript (Sorry if I misunderstood and you were just saying,  "Zareen, come on now, clean that up." :)